Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -437,6 +487,103 @@ badgeCardView config push =
     ] <> FontStyle.body4 TypoGraphy
   ]
 
+driverUpiQrCodeView :: forall w. Config -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w 
+driverUpiQrCodeView config push = 
+  linearLayout 
+    [
+      height WRAP_CONTENT
+    , width MATCH_PARENT
+    , background Color.white900
+    , orientation VERTICAL
+    , cornerRadius 16.0
+    , gravity CENTER
+    , margin $ MarginBottom 8
+    ][linearLayout 
+    [
+      height MATCH_PARENT
+    , width MATCH_PARENT
+    , background Color.yellow800
+    , orientation VERTICAL
+    , gravity CENTER
+    , cornerRadii $ Corners 16.0 true true false false
+    , padding $ PaddingVertical 15 12
+    ]
+    [
+      textView $ [
+        text config.driverUpiQrCard.text
+        , margin $ MarginBottom 8
+      ] <> (FontStyle.body2 TypoGraphy)
+      , linearLayout [
+          height WRAP_CONTENT
+        , width WRAP_CONTENT
+        , background Color.white900
+        , orientation HORIZONTAL
+        , cornerRadius 28.0
+        , stroke $ "1,"<> "#E5E7EB"
```

Comment:
Please check this color, if it is already defined 
If not then It can be simply replaced with `"1,#E5E7EB"`